.github/workflows/editor-only.yml

```diff
@@ -81,7 +81,7 @@ jobs:
   sign-editor-windows:
     if: "!contains(github.event.head_commit.message, 'skip-ci')"
     needs: [build-editor]
-    runs-on: windows-2016
+    runs-on: windows-latest
```

Comment:
They're phasing out these machines.
I'm not sure if using "latest" will cause problems, but I don't think so.
Our executables _should_ use the tool dependencies we've specified, so no new linker dependencies should be introduced.